sdk/python/test_data/pipelines/pipeline_with_string_machine_fields.py

```diff
@@ -0,0 +1,21 @@
+from kfp import dsl
```

Comment:
This file seems to be largely duplicate with [pipeline_with_resource_spec.py](https://github.com/kubeflow/pipelines/blob/581b7e5b7e888a12652a48f81d9fcd3f5e195e37/sdk/python/test_data/pipelines/pipeline_with_resource_spec.py), let consolidate and merge them if necessary.

Reply:
Done, deleted pipeline_with_string_machine_fields